widelands/wl_addons_server
# New addon Fist heroes win

This should be ready now.

Anyway i have trouble to test this in multiplayer lan games:

If i try to cheat and give other players a hero, the other players get no inbox message and by looking at the soldier stats for the other players, they also do not have a hero. Testing scenario is:

* Start two instances of widelands
* create a LAN game in one window
* in the other window join this lan game
* start game
* try to cheat and give another player a hero

In single player games all works as expected. Does cheating prevent something?
